### DISCUSSION TITLE
DropDown- NVDA still narrating Edit read only for the role

### DISCUSSION THREAD

**Discussion by karan759** [primefaces/community#2973] [PrimeReact]
https://github.com/primefaces/primereact/issues/6030
The NVDA is still Narrating Edit only List in NVDA
Its should Narrate : Option clickable options as dropdown open list
![image](https://github.com/user-attachments/assets/4d0522bf-e679-4c13-a9e6-96dfc7d454ae)

**Comment by melloware** [comment]
being tracked here: https://github.com/primefaces/primereact/issues/6971
